testsuite/utils/pom.xml

```diff
@@ -32,6 +32,7 @@
     <properties>
         <maven.compiler.target>1.8</maven.compiler.target>
         <maven.compiler.source>1.8</maven.compiler.source>
+        <keycloak.additionalClasspathElement></keycloak.additionalClasspathElement>
```

Comment:
Needs to be non-empty, otherwise maven fails with NPE when not set on commandline. Set to some non-existent default, e.g. NON_EXISTENT_PATH_OVERRIDE_ON_COMMAND_LINE